examples/boardtest_simpletest.py

```diff
@@ -228,7 +228,7 @@
         NUM_SPACES = len(key)
 
 # Print test results
-for key in TEST_RESULTS:
+for key in TEST_RESULTS.items():
```

Comment:
I think this changes what it does. I believe `items` is a list of key/value tuples. Without it, it is like a list of keys only.

Reply:
Ah ok, so I should replace .items() with .keys()?